### PR TITLE
Email validation updates

### PR DESCRIPTION
**Pull request by st-angelo**
Update email validation formula. Each component of the address should have at least 2 characters. (_ _ @ _ _ . _ _)

### CODE REVIEW

File: packages/pure-validations/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@totalsoft/pure-validations",
-  "version": "5.0.9",
+  "version": "5.0.10",
```
Comment (oncicaradupopovici) [inline packages/pure-validations/package.json]: The version will be auto updated by Lerna publish, please undo this change

File: packages/pure-validations/src/primitiveValidators/index.js [hunk 2]
```diff
@@ -37,7 +37,7 @@ export const atLeastOne = Validator(function atLeastOne(x) {
 });
 
 export const email = Validator(function email(x) {
-  const re = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+  const re = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)+)|([^<>()[\]\\.,;:\s@"]{2,})|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]{2,}\.)+[a-zA-Z]{2,}))$/;
```
Comment (oncicaradupopovici) [inline packages/pure-validations/src/primitiveValidators/index.js:40]: please provide some tests to prove this behaviour, personally I don't understand regex, See https://github.com/osstotalsoft/jsbb/blob/master/packages/pure-validations/src/primitiveValidators/__tests__/primtiveValidators.tests.js

**Review by oncicaradupopovici — CHANGES_REQUESTED** [review]

File: packages/pure-validations/src/primitiveValidators/index.js (hunk 2 above)
Comment (st-angelo) [reply packages/pure-validations/src/primitiveValidators/index.js:40]: Added 4 new tests

**Review by oncicaradupopovici — APPROVED** [review]
Great job! Thank you for your contribution!
